pkg/util/azureclient/applens/applens_client.go

```diff
@@ -61,16 +61,20 @@ func NewClient(endpoint, issuerUrlTemplate, caName, scope string, cred azcore.To
 }
 
 func newPipeline(authPolicy []policy.Policy, options *ClientOptions, issuerUrlTemplate, caName string) (*runtime.Pipeline, error) {
-	var cp *x509.CertPool = nil
```

Comment:
Move it to a narrow scope + delete the "nil" literal as it is already implicit in the declaration.